messages/en.json

```
@@ -54,5 +54,11 @@
   "api_result": {
     "다시 시도해 주세요": "Please try again.",
     "로그인에 실패했습니다": "Google login has failed."
+  },
+  "customer_center": {
```

```suggestion
  "customer_center_page": {
```

중요한 건 아니지만 요건 페이지이니 뒤에 "_page" 붙이는거 어떠세요?
signup_success는 페이지는 아니어서 생략하긴 했었습니다.

추가로, 'api_result' 는 json 순서 상 항상 최하단에 위치하게 하는거 어떠실까요??

---

넵 통일성 있게 _page로 수정할게요!

---

> 추가로, 'api_result' 는 json 순서 상 항상 최하단에 위치하게 하는거 어떠실까요??

요것도 한번 의견+확인 부탁드려용

---

네네 좋습니다! 중간에 껴있는 것 보다 맨 끝에 있는게 확인하기 더 편할 것 같아요!

---

✅ cd706a8541239505b36722cc8ca8f344f315a739